docs/nodes/nodes-library/nodes/Xero/README.md

```diff
@@ -0,0 +1,45 @@
+---
+permalink: /nodes/n8n-nodes-base.xero
+---
+
+# Xero
+
+Xero is a company that offers an online cloud-based SaaS accounting software platform for small and medium-sized businesses.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Xero/README.md).
+:::
+
+## Basic Operations
+
+- Contact
+    - Create a contact
+    - Get a contact
+    - Get all contacts
+    - Update a contact
+- Invoice
+    - Create an invoice
+    - Get an invoice
+    - Get all invoices
+    - Update an invoice
+
+## Example Usage
+
+This workflow allows you to get upto 100 invoices from Xero. You can also find the [workflow](https://n8n.io/workflows/543) on the website. This example usage workflow would use the following two nodes.
```

Comment:
The workflow on n8n.io says 'Get all invoices from Xero'. Could you please make sure that the description here and there are consistent?